server/routerlicious/packages/lambdas/src/test/scribe/lambda.spec.ts

```diff
@@ -7,7 +7,7 @@
 import { ICreateTreeEntry, ICreateTreeParams, ITree } from "@fluidframework/gitresources";
 import { GitManager } from "@fluidframework/server-services-client";
 import { DefaultServiceConfiguration, ICollection, IDocument, IProducer, ITenantManager, MongoManager } from "@fluidframework/server-services-core";
-import { KafkaMessageFactory, MessageFactory, TestCollection, TestContext, TestDbFactory, TestKafka, TestTenantManager } from "@fluidframework/server-test-utils";
+import { KafkaMessageFactory, MessageFactory, TestCollection, TestContext, TestDbFactory, TestDeltaManager, TestKafka, TestTenantManager } from "@fluidframework/server-test-utils";
```

Comment:
Interesting that eslint did not complain about this line length... I guess we don't run esling on test files?

Reply:
Correct. We are explicitly ignoring the spec.ts files

![image](https://user-images.githubusercontent.com/448106/213359724-1e41688c-8c3b-4f54-8f5c-c4b31efdd107.png)